base/linalg/uniformscaling.jl

```diff
@@ -167,6 +167,8 @@ end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
 norm(J::UniformScaling, p::Real=2) = abs(J.λ)
+det(J::UniformScaling{T}) where {T} = one(T)
```

Comment:
Right, but det of  `10*eye(n,n)` == 10^n` not 1.0, so I don't think this will work.